lib/FrontendTool/FrontendTool.cpp

```diff
@@ -322,8 +322,8 @@ static bool atomicallyWritingToTextFile(
 
   bool actionFailed = false;
   std::error_code EC =
-      swift::atomicallyWritingToFile(outputPath, /*binary*/false,
-                                     [&](llvm::raw_pwrite_stream &out) {
```

Comment:
Indentation got messed up here.

Reply:
llvm style I think would say that it should be two. That being said. I am fine fixing it in a subsequent commit.

Reply:
Heh. I looked for it, actually, and they don't say anything about what to do when wrapping lines. I thought clang-format double-indented wrapped lines, but I might be misremembering.